# ms_cgig_chr8_oshv1
Repository to accompany OsHV-1 analyses of CHR8 data, including imputation.      

This work supports the following preprint: (uploading currently)    

Sections:     
- [1. wgrs of OsHV-1 exposure survivors (CHR8 families)](#01-wgrs-of-oshv-1-exposure)    
- [2. Low-resolution GWAS with amplicon panel (OsHV-1 trial)](#02-low-resolution-gwas-with-amplicon-panel-oshv-1-trial)    
- [3. Imputation using amplicon panel and whole-genome resequencing data](#01-wgrs-of-oshv-1-exposure)    

#### Requirements ####
[amplitools](https://github.com/bensutherland/amplitools)      
[simple_pop_stats](https://github.com/bensutherland/simple_pop_stats)      
[wgrs_workflow](https://github.com/bensutherland/wgrs_workflow)       
[impute_workflow](https://github.com/bensutherland/impute_workflow)      
[GEMMA](https://github.com/genetics-statistics/GEMMA/tree/master)       


### 01. WGRS of OsHV-1 exposure ###
A repository has been created for genotyping from whole-genome resequencing data: [wgrs_workflow](https://github.com/bensutherland/wgrs_workflow)        
Use this repository to genotype all samples together, including offspring and parents.     

### 02. Low-resolution GWAS with amplicon panel (OsHV-1 trial) ###
See appendix README: [Low-resolution GWAS with amplicon panel (OsHV-1 trial)](20_docs/README_LR_panel_GWAS.md)     

### 03. Imputation analysis ###
A repository has been created for imputation from the whole-genome resequencing data for parents and low-density amplicon panel genotypes from parents and offspring. It is available here: [impute_workflow](https://github.com/bensutherland/impute_workflow)      
